pootle/apps/pootle_app/views/admin/__init__.py

```diff
@@ -7,6 +7,9 @@
 # or later license. See the LICENSE file for a copy of the license and the
 # AUTHORS file for copyright and authorship information.
 
-from .languages import *
-from .projects import *
-from .users import *
+from .languages import LanguageAdminView, LanguageAPIView
+from .projects import ProjectAdminView, ProjectAPIView
+from .users import UserAdminView, UserAPIView
+
+(LanguageAdminView, LanguageAPIView, ProjectAdminView, ProjectAPIView,
+ UserAdminView, UserAPIView)
```

Comment:
import * is not explicit and is regarded as bad

https://www.python.org/dev/peps/pep-0008/#id17